generate_figures/utilities.py

```diff
@@ -0,0 +1,55 @@
+import pandas as pd
+import wandb
+
+def plot_params(figsize, fontsize, ticksize):
+    params = {
+        "text.usetex": True,
+        "font.family": "serif",
+        "figure.figsize": figsize,
+        "figure.dpi": 80,
+        "figure.edgecolor": "k",
+        "font.size": fontsize,
+        "axes.labelsize": fontsize,
+        "axes.titlesize": fontsize,
+        "xtick.labelsize": ticksize,
+        "ytick.labelsize": ticksize,
+    }
+    return(params)
+
+
+
+def get_results_by_tag(project, tag, test_results = False):
+    api = wandb.Api()
+    count=0
```

Comment:
This whole `count` thing is driving me crazy!  Lets do the following.  Change the signature to
```python
def get_results_by_tag(project, tag, test_results = False, max_runs = 1000):
```

Then get rid of all of the `count` stuff here and I will point out the next thing.

Reply:
@jlperla haha okie yes that is much nicer